haptic_mpc:
    sim3:
        # Skin topic list. Each must publish a TaxelArray message
        skin_list:
          sim: ['/skin/taxel_array', '/haptic_mpc/simulation/proximity/taxel_array']
          none: []

      #  wp_goal_tf_frame: '/world'
        torso_frame: '/torso_lift_link'
        inertial_frame: '/world'

        jep_start: [deg(-100.0), deg(110.0), deg(110.0)]
        planar: true

        # Joint limits, degrees
        joint_limits: 
          max: [150.0, 162.0, 159.0]
          min: [-150.0, -63.0, 0.0]

